### PR TITLE
Update 107_plugins/faqs

### PR DESCRIPTION
**Pull request by yesszus**
Removed errors.

https://github.com/e107inc/e107/pull/2511#issuecomment-292855614

### CODE REVIEW

File: e107_languages/English/English.php [hunk 1]
```diff
@@ -141,4 +141,13 @@
...
 
+//FAQ
+define("LANA_FAQ_CRON_1", "Unanswered Questions Report");
+define("LANA_FAQ_CRON_2", "Mails a report of unanswered questions to [x].");
+define("LANA_FAQ_CRON_3", "[x] Unanswered Questions at [y]");
+define("LANA_FAQ_CRON_4", "Please login to your site to review and answer these questions using the link below.");
+define("LANA_FAQ_CRON_5", "The [x] most recent questions are displayed below.");
+define("LANA_FAQ_CRON_6", "[x] Unanswered Question as of [y].");
+define("LAN_CLICK_HERE", "CLICK HERE");
+
```
Comment (CaMer0n) [inline e107_languages/English/English.php]: Normally we don't add FAQ LANS to e107's core language file.

Comment (yesszus) [inline e107_languages/English/English.php]: Ok. Restored.

**Comment by yesszus** [comment]
@CaMer0n updated this.

File: e107_plugins/faqs/faqs_shortcodes.php [hunk 2]
```diff
@@ -298,7 +298,7 @@ function sc_faq_submit_question($parms)
 				$text .= $frm->open('faq-ask-question','post');
 				//TODO LAN ie. [x] character limit.
 				$text .= "<div>".$frm->textarea('ask_a_question','',3, 80 ,array('maxlength'=>$this->questionCharLimit, 'size'=>'xxlarge','placeholder'=>LAN_FAQS_ENTER_A_QUESTION_HERE, 'wrap'=>'soft'))."
-				<div class='faq-char-limit'><small>".$this->questionCharLimit." ".LAN_FAQS_CHAR_LIMIT".</small></div>".$frm->submit('submit_a_question',LAN_SUBMIT)."</div>";
+				<div class='faq-char-limit'><small>".$this->questionCharLimit.LAN_FAQS_CHAR_LIMIT".</small></div>".$frm->submit('submit_a_question',LAN_SUBMIT)."</div>";
```
Comment (CaMer0n) [inline e107_plugins/faqs/faqs_shortcodes.php]: So the space is no longer needed?

Comment (yesszus) [inline e107_plugins/faqs/faqs_shortcodes.php]: @CaMer0n work this and changed.

**Pull request closed without merging**